lib/lure/repo.go

```diff
@@ -91,36 +91,56 @@ func checkForUpdatesJob(auth Authentication, project Project) error {
 		updateModule(auth, moduleToUpdate, project, repo, pullRequests)
 	}
 
+	log.Printf("Info: Check for updates done.")
+
 	return nil
 }
 
 func updateModule(auth Authentication, moduleToUpdate moduleVersion, project Project, repo Repo, existingPRs []PullRequest) {
-	var title string
 	var dependencyName string
 	if moduleToUpdate.Name != "" {
 		dependencyName = moduleToUpdate.Name
 	} else {
 		dependencyName = moduleToUpdate.Module
 	}
-	title = fmt.Sprintf("Update %s dependency %s to version %s", moduleToUpdate.Type, dependencyName, moduleToUpdate.Latest)
+
+	title := fmt.Sprintf("Update %s dependency %s to version %s", moduleToUpdate.Type, dependencyName, moduleToUpdate.Latest)
+
+	branchPrefix := project.BranchPrefix
+	if branchPrefix == "" {
+		branchPrefix = "lure-"
+	}
+	dependencyBranchPrefix := HgSanitizeBranchName(branchPrefix + dependencyName)
+	dependencyBranchVersionPrefix := dependencyBranchPrefix + "-" + HgSanitizeBranchName(moduleToUpdate.Latest)
+	branchGUID, _ := guid.V4()
+	var branch = dependencyBranchVersionPrefix + "-" + branchGUID.String()
+
+	var prAlreadyExists = false
 	for _, pr := range existingPRs {
-		if pr.Title == title {
-			log.Printf("There already is a PR for: %s", title)
-			return
+		if strings.HasPrefix(pr.Source.Branch.Name, dependencyBranchVersionPrefix) {
+			log.Printf("There already is a PR for: %s, branch name is: %s.", title, pr.Source.Branch.Name)
```

Comment:
"There already is a PR for: %s. The branch name is: %s."
Je pense qu'on devrait quoter le titre.